### ISSUE TITLE
add_risktable y label gets moved 

### ISSUE THREAD

**Issue by SHAESEN2** [openpharma/visR#272]
When adding the risktable to the plot, the y label moves entirely to the left of the plot, leaving a big gap between the Y-label and the plot. This should be adressed.

**Comment by SHAESEN2** [comment]
Hey Tim,

You only see it when the labels are longer.

``` r
library(visR)

surv <- estimate_KM(adtte, strata=c("SEX", "TRTP"))

visr(surv) %>%
  add_risktable(group="statlist")
  
```

![image](https://user-images.githubusercontent.com/47894155/139635588-ec4ef90d-43f5-4f95-88e8-802f67710b49.png)

**Comment by timtreis** [comment]
Thanks Steven, could reproduce!

Debugging WIP:
Issue most likely caused because `visR:::align_plots()` aligns the gtable with the x-axis positions. The strata-names are formally tick labels? Therefore, the y-label of the upper plot should be to the left of the tick labels and gets moved.

**Comment by timtreis** [comment]
I think I managed to fix it, do you want to try to break it @SHAESEN2?

![image](https://user-images.githubusercontent.com/12818242/139680563-2f90abb3-2baf-4a2a-a1be-cc4663e1073d.png)

**Comment by timtreis** [comment]
How relevant is this issue? The only actually robust solutions I see here are massively overengineered for what little benefit it brings. By fixing this, we're violating the underlying grid principle since we'd want to place a component outside of his designated square. This is calculated on the fly while rendering (afaik) so it'd be a gigantic mess (worth a lot of hours of code) to even find a somewhat robust solution.

**Comment by SHAESEN2** [comment]
It should not be that difficult. Cowplot allows the label to be aligned by default between two plots or even allows it to be shifted to the most left side of the complete figure. Could you check their solution and see if we can implement similarly?

**Comment by timtreis** [comment]
Don't really want to. I already poured 6+ hours into this and would rather get the table two and the auto-deploy of our interactive vignette to shinyapps.io running :D Maybe I'll check in with this later

**Comment by SHAESEN2** [comment]
Understood. I will reassign the issue to myself.

**Comment by timtreis** [comment]
Thanks!

**Comment by SHAESEN2** [comment]
#294

**Comment by SHAESEN2** [comment]
Closed with #294
